app/models/chef.rb

```diff
@@ -1,4 +1,10 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :dish_ingredients, through: :dishes
+   has_many :ingredients, through: :dish_ingredients
+
+   def unique_ingredients
+      ingredients.distinct
```

Comment:
Excellent work for US 3! Adding these additional associations above make this query a lot easier. Nice work!